client/src/modules/webhooks/Webhooks.tsx

```diff
@@ -48,7 +49,7 @@ export const Webhooks = ({ appId }: AppDomainProps) => {
         <GridItem colSpan={8} pt={3}>
           <Text>
             {isSecretVisible
-              ? data.app?.appMetaGithub?.webhooksSecret
+              ? config.githubWebhooksSecret
```

Comment:
@Akirtovskis I think we can remove this totally, the user should not be able too see it and change it.